Modules/IO/NIFTI/include/itkNiftiImageIO.h

```diff
@@ -238,6 +269,10 @@ class ITKIONIFTI_EXPORT NiftiImageIO : public ImageIOBase
   double m_RescaleSlope{ 1.0 };
   double m_RescaleIntercept{ 0.0 };
 
+  bool m_ConvertRAS{ false };
```

Comment:
Private members also need to get printed as per the ITK SW Guide:
https://github.com/InsightSoftwareConsortium/ITKSoftwareGuide/blob/master/SoftwareGuide/Latex/Appendices/CodingStyleGuide.tex#L3622

Reply:
OK, I'll print it.